ui/packages/catalog-ui-search/src/main/webapp/js/ApplicationSetup.js

```diff
@@ -158,5 +158,5 @@ require([
     };
 
     require('js/ApplicationStart');
-    require('codice-icons/icons/codice.font');
+    require('codice-icons/codice.font.config');
```

Comment:
❓ Why the update?

Reply:
At some point, this was breaking because it couldn't find `codice-icons/icons/codice.font`. Not sure why that was, but it's definitely there now. I'll change it back.